docs/pages/reference/metrics.mdx

```diff
@@ -83,75 +83,86 @@ Teleport Cloud does not expose monitoring endpoints for the Auth Service and Pro
+| `backend_write_seconds`                         | histogram   | cache               | Latency for backend write operations.                                                              |
+| `cluster_name_not_found_total`                  | counter     | Teleport Auth       | Number of times a cluster was not found.                                                           |
+| `dynamo_requests_total`                         | counter     | DynamoDB            | Number of requests to the DynamoDB api.                                                            |
+| `dynamo_requests`                               | counter     | DynamoDB            | Number of failed requests to the DynamoDB api.                                                     |
+| `dynamo_requests_seconds`                       | histogram   | DynamoDB            | Latency of DynamoDB API requests.                                                                  |
+| `dynamo_requests_total`                         | counter     | DynamoDB            | Number of requests to the DynamoDB API                                                             |
+| `etcd_backend_batch_read_requests`              | counter     | etcd                | Number of read requests to the etcd database.                                                      |
+| `etcd_backend_batch_read_seconds`               | histogram   | etcd                | Latency for etcd read operations.                                                                  |
+| `etcd_backend_read_requests`                    | counter     | etcd                | Number of read requests to the etcd database.                                                      |
+| `etcd_backend_read_seconds`                     | histogram   | etcd                | Latency for etcd read operations.                                                                  |
+| `etcd_backend_tx_requests`                      | counter     | etcd                | Number of transaction requests to the database.                                                    |
+| `etcd_backend_tx_seconds`                       | histogram   | etcd                | Latency for etcd transaction operations.                                                           |
+| `etcd_backend_write_requests`                   | counter     | etcd                | Number of write requests to the database.                                                          |
+| `etcd_backend_write_seconds`                    | histogram   | etcd                | Latency for etcd write operations.                                                                 |
+| `firestore_events_backend_batch_read_requests`  | counter     | GCP Cloud Firestore | Number of batch read requests to Cloud Firestore events.                                           |
```

Comment:
Should "GCP Cloud Firestore" (or just "Cloud Firestore") be "Google Cloud Firestore" here and below?

Reply:
Googling for "Google Cloud Firestore" and "GCP Cloud Firestore" both bring up the same results, while just "Cloud Firestore" bring up a less relevant first hit. Between "Google" and "GCP", I think we can trust the audience of this page to know that acronym, and tables like this are the place to be concise.